app/services/processing_service.py

```
@@ -412,6 +431,22 @@ async def process_repository(self, job_payload: Dict[str, Any]) -> ProcessingRes
                 total_chunks=len(chunks),
                 total_embeddings=len(embeddings),
             )
+            
+            qna_pkg = None
+            try:
+                logger.info("Generating repository Q & A summary .... ")
+                qna_pkg = await generate_project_qna(
```

Ruff (0.12.2)

Local variable `qna_pkg` is assigned to but never used

Remove assignment to unused variable `qna_pkg`

(F841)